src/editors/containers/VideoEditor/components/VideoSettingsModal/components/ThumbnailWidget/utils.js

```diff
@@ -0,0 +1,11 @@
+import { StrictDict } from '../../../../../../utils';
+
+export const acceptedImgKeys = StrictDict({
```

Comment:
this could go in /data/constants as well but I think here is fine (you might call this file constants)

Reply:
will do in resampling PR as I will be adding more variables to the file